Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -100,35 +101,39 @@ def get_classification(self, image):
         
         image_np = np.expand_dims(np.asarray(image, dtype=np.uint8), 0)
         
-        with tf.Session(graph=detection_graph) as sess:                
+        with tf.Session(graph=self.detection_graph) as sess:                
             # Actual detection.
-            (boxes, scores, classes) = sess.run([detection_boxes, detection_scores, detection_classes], 
-                                        feed_dict={image_tensor: image_np})
+            (boxes, scores, classes) = sess.run([self.detection_boxes, self.detection_scores, self.detection_classes], 
+                                        feed_dict={self.image_tensor: image_np})
 
         boxes = np.squeeze(boxes)
         scores = np.squeeze(scores)
         classes = np.squeeze(classes)
-        confidence_cutoff = 0.5
+        confidence_cutoff = 0.1
         
         # Filter boxes with a confidence score less than `confidence_cutoff`
-        final_boxes, final_scores, final_classes = filter_boxes(confidence_cutoff, boxes, scores, classes)
-        
+        final_boxes, final_scores, final_classes = self.filter_boxes(confidence_cutoff, boxes, scores, classes)
         # Enable for Visualization
-        #width, height = image.size
-        #box_coords = to_image_coords(final_boxes, height, width)
-        #draw_boxes(image, box_coords, classes)
-        
+        if (self.enable_visualization):
```

Comment:
No need to have left and right parenthesis in if statement

Reply:
fixed